src/main/java/com/jfrog/ide/idea/scan/ScanBinaryExecutor.java

```diff
@@ -151,24 +165,32 @@ protected List<JFrogSecurityWarning> execute(ScanConfig.Builder inputFileBuilder
             Logger log = Logger.getInstance();
             // The following logging is done outside the commandExecutor because the commandExecutor log level is set to INFO.
             //  As it is an internal binary execution, the message should be printed for DEBUG use only.
-            log.debug(String.format("Executing command: %s %s", binaryTargetPath.toString(), join(" ", args)));
+            indicator.setText(String.format("Running %s scan at %s", scanType.toString().toLowerCase(), String.join(" ", inputParams.getRoots())));
+            String cmd = String.format("%s %s", binaryTargetPath.toString(), join(" ", args));
+            log.info(String.format("""
+                    Executing JAS scanner
+                    %s
+                    with config:
+                    %s""", cmd, inputParams));
             CommandExecutor commandExecutor = new CommandExecutor(binaryTargetPath.toString(), createEnvWithCredentials());
             CommandResults commandResults = commandExecutor.exeCommand(binaryTargetPath.toFile().getParentFile(), args,
-                    null, new NullLog(), MAX_EXECUTION_MINUTES, TimeUnit.MINUTES);
+                    null, new NullLog(), Long.MAX_VALUE, TimeUnit.MINUTES);
 
             checkCanceled.run();
 
             if (commandResults.isOk()) {
+                log.info(String.format("Finished successfully to run command: %s", cmd));
                 log.debug(commandResults.getRes());
                 return parseOutputSarif(outputFilePath);
             }
+            log.info(String.format("Failed to run command: %s", cmd));
             switch (commandResults.getExitValue()) {
                 case USER_NOT_ENTITLED -> {
-                    log.debug("User not entitled for advance security scan");
+                    log.info("User not entitled for advance security scan");
```

Comment:
I'm not sure we should log this for users who aren't entitled to advanced security